src/Command/InfectionCommand.php

```diff
@@ -202,26 +208,24 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $container = $this->getContainer();
-
-        if (!$container->get('coverage.checker')->hasDebuggerOrCoverageOption()) {
+        if (!$this->container['coverage.checker']->hasDebuggerOrCoverageOption()) {
             $this->consoleOutput->logMissedDebuggerOrCoverageOption();
 
             return 1;
         }
 
         /** @var InfectionConfig $config */
-        $config = $container->get('infection.config');
+        $config = $this->container['infection.config'];
```

Comment:
>If we no longer use that container, then we can also drop the dependency

That's the plan although I don't know if it can be done in that PR yet;

>I personally prefer to use the PSR container, but i'm okay with dropping that.

Well from the internal discussions we are moving towards typehinted getters, so PSR11 doesn't provide much there; We are still using a typehint-less way here simply because it would be too big of a change